app/views/how-tos/making-pages.html

```diff
@@ -17,29 +17,42 @@ <h1>
         {{ pageName }}
       </h1>
 
-      <p class="nhsuk-lede-text">Use your HTML text editor to make pages.</p>
+      <p class="nhsuk-lede-text">Use your code editor to make pages.</p>
 
-      <p>Create all your HTML pages in the <strong>/app/views</strong> folder.</p>
+      <p>Create all your pages in the <code>/app/views</code> folder.</p>
 
-      <p>All pages need the <strong>.html</strong> extension.</p>
+      <p>The file names must end in <code>.html</code> – this is called the file extension.</p>
 
-      <p>Pages within <strong>/app/views</strong> will be automatically available in your web browser.</p>
+      <p> It’s a good idea to name the files using a lowercase version of the page title, with spaces replaced by hyphens. For example, <code>find-a-gap.html</code>.</p>
 
-      <p>For example, if you create a page called <strong>contact.html</strong> and then go to <strong>localhost:3000/contact</strong> in your browser, you will see that page.</p>
+      <p>The pages you create are automatically viewable in your browser at the path that matches the file name. For example, if you create a page called <code>contact.html</code> and go to <code>http://localhost:3000/contact</code> in your browser, you will see that page.</p>
 
-      <img class="app-img-guide" alt="Screenshot of creating a new page" src="/images/page-example.png" />
+      <h2>Specifying a layout</h2>
 
-      <h2>Using folders</h2>
+      <p>Each page you create will need to specify which layout it uses. Otherwise you’ll likely get a plain white background with no header or footer!</p>
```

Comment:
```suggestion
      <p>Each page you create will need to specify which layout it uses. Otherwise you’ll will get white background with no header or footer!</p>
```